### ISSUE TITLE
`local -x` is not recognized as `export` or `declare -x` (bug)

### ISSUE THREAD

**Issue by IsaacG** [koalaman/shellcheck#2069]
#### For bugs
- Rule Id: SC2034
- My shellcheck version: online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

f () {
        local -x a
        # export a
        a=1
        cmd
}
```

#### Here's what shellcheck currently says:
```
$ shellcheck myscript
 
Line 6:
        a=1
        ^-- SC2034: a appears unused. Verify use (or export if used externally).
```
#### Here's what I wanted or expected to see:

`export a` or `declare -x a` makes shellcheck recognize `a` as exported and makes SC2034 go away. `local -x` is not recognized the same way. `local -x` should have the same effect as `declare -x`

**Comment by koalaman** [comment]
Indeed it should, and now it does. Thanks!
